src/frontend/komponenter/Skjemasteg/Skjemasteg.tsx

```diff
@@ -79,12 +79,12 @@ const Skjemasteg: React.FunctionComponent<IProps> = ({
     const location = useLocation();
     const {
         forrigeÅpneSide,
-        åpenBehandling,
+        behandling,
         vurderErLesevisning,
         erBehandleneEnhetMidlertidig,
         erBehandlingAvsluttet,
     } = useBehandlingContext();
-    const erBehandlingSattPåVent = hentDataFraRessurs(åpenBehandling)?.behandlingPåVent;
+    const erBehandlingSattPåVent = behandling?.behandlingPåVent;
```

Comment:
Tror ikke du trenger `?`

```suggestion
    const erBehandlingSattPåVent = behandling.behandlingPåVent;
```